client/src/components/relatedItems/relatedCarousel/relatedCard/cardComponents/compareComponents/featuresCheck.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+function FeaturesCheck({ feature, currentFeatures }) {
+  let featureValue;
+
+  for (let i = 0; i < currentFeatures.length; i++) {
+    if (currentFeatures[i].feature === feature) {
+      featureValue = currentFeatures[i].value;
+      break;
+    }
+    featureValue = 'no such feature';
+  }
+
+  if (featureValue === true) {
+    return (
+      <>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+        <td><i className="fa fa-check"></i></td>
+      </>
+    );
+  } if (featureValue === null) {
```

Comment:
More of a suggestion, you do not need to change this if you don't want to! I believe you can use a conditional ternary operator for lines 21 -31.  Since null is falsy, you can do something like featureValue ? return lines 24-27 : return line 30.

Reply:
i specifically needed to check for null so i can return an X because some of the features had a null value and the feature property. meaning the product line have that feature but that specific product does not have that value, whereas on line 30 i just return empty element when the product doesnt have a feature property of that feature.